backend/clickhouse/logs.go

```diff
@@ -2,91 +2,36 @@ package clickhouse
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
-	"strconv"
 	"time"
 
 	modelInputs "github.com/highlight-run/highlight/backend/private-graph/graph/model"
 	e "github.com/pkg/errors"
-	log "github.com/sirupsen/logrus"
 )
 
 type LogRow struct {
-	Timestamp       time.Time
-	SeverityText    string
-	Body            string
-	ProjectId       uint32
-	SecureSessionID string
+	Timestamp          time.Time
+	TraceId            string
+	SpanId             string
+	TraceFlags         uint32
+	SeverityText       string
+	SeverityNumber     int32
+	ServiceName        string
+	Body               string
+	ResourceAttributes map[string]string
+	LogAttributes      map[string]string
+	ProjectId          uint32
+	SecureSessionId    string
 }
 
-const LogsTable = "logs"
-
-// func (client *Client) CreateLogsTable(ctx context.Context) error {
-// 	client.conn.Exec(ctx, "DROP TABLE IF EXISTS logs") //nolint:errcheck
-
-// 	return client.conn.Exec(ctx, `
-// 	CREATE TABLE IF NOT EXISTS logs (
-// 		Timestamp       DateTime64(9) CODEC (Delta, ZSTD(1)),
-// 		SeverityText    LowCardinality(String) CODEC (ZSTD(1)),
-// 		Body            String CODEC (ZSTD(1)),
-// 		ProjectId 		UInt32 CODEC (ZSTD(1)),
-// 		SecureSessionID Nullable(String) CODEC (ZSTD(1))
-// 	)
-// 	ENGINE = MergeTree()
-// 		PARTITION BY toDate(Timestamp)
-// 		ORDER BY (ProjectId, toUnixTimestamp(Timestamp))
-// 		TTL toDateTime(Timestamp) + toIntervalDay(30)
-// 		SETTINGS index_granularity = 8192, ttl_only_drop_parts = 1;
-// 	`)
-// }
-
-type Message struct {
-	Type  string `json:"type"`
-	Trace []struct {
-		ColumnNumber int    `json:"columnNumber"`
-		LineNumber   int    `json:"lineNumber"`
-		FileName     string `json:"fileName"`
-		FunctionName string `json:"functionName,omitempty"`
-		Source       string `json:"source"`
-	} `json:"trace"`
-	Value []string `json:"value"`
-	Time  int64    `json:"time"`
-}
-
-type Messages struct {
-	Messages []Message `json:"messages"`
-}
-
-func ParseConsoleMessages(projectID int, sessionSecureID string, messages string) ([]*LogRow, error) {
```

Comment:
I moved this to `console_messages_parser.go` to keep this top level interface light weight. If we add additional parsing for other sdks, they can go in a respective file (e.g. `python_logs_parser.go`.